examples/collection-only/collection.json

```diff
@@ -1,7 +1,7 @@
 {
   "stac_version": "1.0.0-beta.2",
   "stac_extensions": [],
-  "id": "COPERNICUS/S2",
+  "id": "collection-only",
```

Comment:
The ids are a bit strange here.

Reply:
What would you have them be instead?

Reply:
Maybe `sentinel-2`?

Reply:
Ah, cool. I thought you were saying all the example ids were strange. Will update this one.